service/src/main/kotlin/fi/espoo/evaka/invoicing/domain/Payments.kt

```diff
@@ -32,10 +32,10 @@ interface PaymentIntegrationClient {
         val failed: List<Payment> = listOf()
     )
 
-    fun send(payments: List<Payment>): SendResult
+    fun send(payments: List<Payment>, tx: Database.Transaction): SendResult
```

Comment:
```suggestion
    fun send(payments: List<Payment>, tx: Database.Read): SendResult
```

If there is no need to run updates to database (`Transaction` extends `Read`).